FormattingStyleGuide.md

```diff
@@ -0,0 +1,43 @@
+# Formatting & Style Guide
+
+## Scope and purpose
+This document provides formatting and style conventions for publishing documents in __GitHub__ in __Markdown__ format. Some advice on document conversion and how to meet more sophisticated publishing needs is offered.
+
+## General
+- Use [GitHub Flavored Markdown](https://guides.github.com/features/mastering-markdown/).
+- If the workflow prescribes the document to go into BitBucket repository first, then you may find [BitBucket Markdown Syntax Guide](https://confluence.atlassian.com/bitbucketserver/markdown-syntax-guide-776639995.html) useful.
+
+## Title & headings
+- Use # (h1) for document title.
+- Use ## (h2) for top-level headings.
+- Number the headings manually. There's no heading numbering support in Markdown.
+
+## Table of contents
+- Provide table of contents for longer documents.
+- Use links in the table of contents. Use [GitHub heading anchors](https://gist.github.com/asabaylus/3071099). For example,
+```
+  [Table of contents](StyleManual.md#table-of-contents) 
+```
+links to this section.
+
+## Linking
+- Use [relative linking)(https://github.com/blog/1395-relative-links-in-markup-files) to link to image files and other documents in the same repository.
+
+## Code
+- Use backticks to mark inline code, URLs a.o. constant values.
+- Use GitHub Markdown code fencing to format blocks of code.
+
+## Tables
+- Use GitHub Flavored Markdown features.
+
+## Conversion
+- Use programmer's editor (e.g. [Sublime Text](https://www.sublimetext.com/)), or just Notepad to convert documents from Word, Confluence, PDF a.o. formats to Markdown.
+- Specialised converter software (e.g. [Pandoc](http://pandoc.org/)) can be used, but will rarely justify the learning effort.
```

Comment:
I started the conversion using pandoc and continued editing manually. At least with a document with a lot of tables, pandoc is a good starting point. I don't know if any editor can copy tables to md.

I used the following command. The option -t markdown_github is important to get the tables right.
pandoc --standalone --smart  xtee6/doc/Manuals/ug-syspar_x-road_v6_system_parameters.odt_2.9_Y-881-1.docx -t markdown_github -o ug-syspar_x-road_v6_system_parameters.md

After this command, the document still needs quite a bit of manual editing, especially with multi line tables. For editing, I used MD editor haroopad.